corehq/motech/openmrs/tasks.py

```diff
@@ -200,8 +201,8 @@ def import_patients_to_domain(domain_name, force=False):
                 location_type = LocationType.objects.get(domain=domain_name, name=importer.location_type_name)
             except LocationType.DoesNotExist:
                 logger.error(
-                    'No organization level named "{location_type}" found in project space "{domain}".'.format(
-                        location_type=importer.location_type_name, domain=domain_name)
+                    f'No organization level named "{importer.location_type_name}" found in project space '
+                    f'"{domain_name}".'
```

Comment:
could include this on the line above

Reply:
Yeah, I stuck with the 115-character line length of the existing code, but I agree that these dangling bits at the end look dumb. I'll clean them up.